pkg/cac/cac.go

```diff
@@ -7,86 +7,89 @@ package cac
 import (
 	"bytes"
 	"encoding/binary"
-	"errors"
+	"fmt"
 
 	"github.com/ethersphere/bee/pkg/bmtpool"
 	"github.com/ethersphere/bee/pkg/swarm"
 )
 
 var (
-	errTooShortChunkData = errors.New("short chunk data")
-	errTooLargeChunkData = errors.New("data too large")
+	ErrChunkSpanShort = fmt.Errorf("chunk span must have exactly length of %d", swarm.SpanSize)
+	ErrChunkDataLarge = fmt.Errorf("chunk data exceeds maximum allowed length")
```

Comment:
this could be left out as `errors.New` since no formatting is happening, but since it's compile time might as well leave it like this.